app/2.0.x/db-less-admin-api.md

```diff
@@ -1,6 +1,12 @@
 ---
+#
+#  WARNING: this file was auto-generated by a script.
+#  DO NOT edit this file directly. Instead, send a pull request to change
+#  https://github.com/Kong/kong/blob/master/scripts/autodoc-admin-api
+#  or its associated files instead.
+#
 title: Admin API for DB-less Mode
-skip_read_time: true
+toc: false
```

Comment:
The db-less version of this doc is made from the same template as the Admin API version, which includes this bit. So I had to add it here as well.